swagger.yaml

```diff
@@ -0,0 +1,91 @@
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: My Tutorial project!!!
+host: localhost:8000
+tags:
+- description: API for design tutorial
+schemes:
+- http
+consumes:
+- application/json
+produces:
+- application/json
+paths:
+  "/Tutorial":
+    parameters:
+    - name: id
+      in: path
+      required: true
+      description: ID of tutorial that we want to find
+      type: integer
+    get:
+      tags:
+      - Tutorial
+      summary: Get all tutorial in system
+      responses:
+        '200':
+          description: Tutorial data
+          schema:
+            "$ref": "#/definitions/Tutorial"
+    post:
+      summary: " create tutorial in system"
+      tags:
+      - Tutorial
+      description: Add new tutorial in system
+      parameters:
+      - name: tutorial
+        in: body
+        schema:
+          "$ref": "#/definitions/Tutorial"
+      produces:
+      - application/json
+      responses:
+        '200':
+          description: New tutorial is created
+          schema:
+            "$ref": "#/definitions/Tutorial"
+    delete:
+      summary: Delete tutorial with given ID
+      tags:
+      - Tutorial
+      responses:
+        '200':
+          description: tutorial is deleted!!
+          schema:
+            "$ref": "#/definitions/Tutorial"
+    put:
```

Comment:
We should always use `patch` method. Use PUT only it's really required.
https://www.baeldung.com/http-put-patch-difference-spring